Tests/DependencyInjection/DoctrineMigrationsExtensionTest.php

```diff
@@ -285,6 +287,10 @@ public function testPrefersEntityManagerOverConnection(): void
             'migrations_paths' => ['DoctrineMigrationsTest' => 'a'],
         ];
         $ormConfig = trait_exists(LazyGhostTrait::class) ? ['enable_lazy_ghost_objects' => true] : [];
+        if (PHP_VERSION_ID >= 73000) {
+            $ormConfig['controller_resolver'] = ['auto_mapping' => false];
+        }
```

Comment:
This looks like it would break if I did a low-deps run on PHP 7.3. 🤔

Reply:
It would, I couldn't find a reliable way to detect the right version of the bundle. Since this detection code only occurs in tests, I thought it would not be a big deal.

Reply:
Shall we try `Composer\InstalledVersions` instead then?